Add groupby template variables options (#30834)
DataDog/documentation: content/en/dashboards/template_variables.md

## Overview

> Template variables allow you to dynamically filter one or more widgets in a dashboard. You can build saved views from your template variable selections to organize and navigate your visualizations through the dropdown selections.

Author: ```suggestion
Template variables allow you to dynamically filter or group widgets in a dashboard. You can build saved views from your template variable selections to organize and navigate your visualizations through the dropdown selections.
```

**Note**: If you do not see the tag or attribute you're looking for, it may be because that data hasn't been reported to Datadog recently. For more information, see [Historical Data][4].

## Add a template variable

> If template variables are already defined, see [Edit a template variable](#edit-a-template-variabl). If your dashboard does not have any template variables, click the question mark icon to open a help modal on how to use Dashboard Variables.

Reviewer: ```suggestion
If template variables are already defined, see [Edit a template variable](#edit-a-template-variable). If your dashboard does not have any template variables, click the question mark icon to open a help modal on how to use Dashboard Variables.
```

> If template variables are already defined, see [Edit a template variable](#edit-a-template-variable). If your dashboard does not have any template variables, click the question mark icon to open a help modal on how to use Dashboard Variables.

Author: ```suggestion
If template variables are already defined, see [Edit a template variable](#edit-a-template-variable). If your dashboard does not have any template variables, you can click the question mark icon to open a help modal on how to use Dashboard Variables.
```

### Widgets

> When creating or editing a widget, existing template variables display as options in the `from` field. For example, if you configure the `environment` template variable, the `$environment` option is available as a dynamic variable in the widget.

Author: ```suggestion
When creating or editing a widget, existing filter template variables display as options in the `from` field, and existing group by template variables display as options following the `by` field. For example, if you configure the `environment` template variable, the `$environment` option is available as a dynamic variable in the widget.
```
